README.md

```diff
@@ -44,6 +44,7 @@ This generator can also be further configured with the following command line fl
     -V, --version       output the version number
     -e, --ejs           add ejs engine support (defaults to jade)
         --hbs           add handlebars engine support
+        --static        add support for static html support
```

Comment:
I think something like "use static html without templates" would be a more readable description. Also, can we add the `-s` shortcut?